### DISCUSSION TITLE
Hw0-4 is graded

### DISCUSSION THREAD

**Discussion by spearo2** [prosyslab-classroom/cs424-program-reasoning#465] [Announcements]
안녕하세요.

Hw0-4가 체점 되었습니다. 성적은 GradeScope 에 가면 볼 수 있습니다. 3점 만점입니다. 

1점 이상의 점수(양수)를 받은 학생들은 본인의 멋진 생각이 담긴 글을 모두에게 공유해주길 바랍니다. 아래 절차에 따라서 제출해 주세요.
- 교수님께서 남긴 평을 읽어보고 수정 요청이 적혀있으면 그에 맞게 수정해주세요.
- 그 이외에도 본인이 수정하고 싶은 부분이 있으면 수정해도 됩니다.  학번을 적은 학생은 지우고 제출해주세요.
- 지난 학기 학생들의 제출물을 보고 같은 형식으로 제출(pull request)해 주세요.
  - 이 저장소의 essay 디렉토리 아래에, "2024-name.pdf" 추가
  - [이 페이지](https://github.com/prosyslab-classroom/cs424-program-reasoning/blob/main/hof.md)에 본인의 이름과 파일 링크 추가

감사합니다.

---

Hello,

Hw0-4 is graded. You can check your grades on GradeScope, with a maximum of 3 points. 

Students who received positive points (greater than or equal to 1), please share your excellent ideas with everyone. Follow the steps below to submit your work:
- Read the professor's comments and revise your essay accordingly if any revision requests were mentioned.
- You can also make any other changes you want. Please remove your student ID number before submitting.
- Submit your essay in the same format as the submissions from last semester.
  - Add a file named “2024-name.pdf” under the essay directory of this repository.
  - Add your name and a link to your file to [this page](https://github.com/prosyslab-classroom/cs424-program-reasoning/blob/main/hof.md).

Thanks for all your hard work.

**Comment by JustinKim98** [comment]
I tried to upload my essay and write a pull request, but permission was denied. Is there any other way?
